(DRAFT) Developer site

nielsvanvelzen opened the pull request:
Opening a pull request for visibility and CF pages preview deployment. This is far from ready to merge.

nielsvanvelzen committed:
Create new "developers" docs section

nielsvanvelzen committed:
Move current developer related documentation

nielsvanvelzen committed:
Add documentation for API authorization to developers site

nielsvanvelzen committed:
Update title of authorization page

nielsvanvelzen committed:
Update header navigation

nielsvanvelzen committed:
Replace @docusaurus/preset-classic with manual plugin configurations

nielsvanvelzen committed:
Redo folder structure and paths for developer site, add blog/pages/docs plugins

nielsvanvelzen committed:
Add redirects for developer documents moved from main docs

nielsvanvelzen committed:
Add index page for developers site

nielsvanvelzen committed:
Fix missing new line

nielsvanvelzen committed:
Merge branch 'master' into developers

nielsvanvelzen committed:
Update authorization documentation to latest gist revision

PeachesMLG committed:
Added a work in progress banner under the /developers page

nielsvanvelzen committed:
Reformat + update text

nielsvanvelzen committed:
Update CODEOWNERS

PeachesMLG committed:
Refactor code to reduce duplication (#895)

* Refactor code to reduce duplication

* Reformat using prettier

nielsvanvelzen committed:
Merge branch 'master' into developers

Gauvino commented on content/developers/docs/api/authorization.md:
```diff
@@ -0,0 +1,72 @@
+
+To start using the Jellyfin API, authorization is probably the first thing you'll need to do. Jellyfin's authorization options can be a bit confusing because there are a lot of deprecated options.
+
+Generally there are three ways to authenticate: no authorization, user authorization with an access token or authorization with an API key. The first way is easy, just do nothing. But most often you'll need to use either the access token or API key.
```
```suggestion
Generally, there are three ways to authenticate: no authorization, user authorization with an access token or authorization with an API key. The first way is easy, just do nothing. But most often you'll need to use either the access token or API key.
```

joshuaboniface replied:
Grammatically a comma isn't needed here.

Gauvino reviewed (commented):
little something 🫡

nielsvanvelzen commented:
Changes for this feature branch need to be made in separate pull requests targeting the branch.

anultravioletaurora reviewed (approved):
There is SO much helpful stuff in here 🙏 

Thank you for putting this together, Niels!

nielsvanvelzen committed:
Merge branch 'master' into developers

nielsvanvelzen committed:
Fix bad merge

anultravioletaurora committed:
Create syncplay.md

anultravioletaurora committed:
Fix build

anultravioletaurora committed:
Fix link on clients index list

nielsvanvelzen committed:
Merge branch 'master' into developers

anultravioletaurora committed:
Merge branch 'developers' into developers

anultravioletaurora committed:
fix moar broken links

anultravioletaurora committed:
Merge branch 'developers' of github.com:anultravioletaurora/jellyfin.org into developers

nielsvanvelzen committed:
Merge pull request #1550 from anultravioletaurora/developers

Fix Developer Documentation Branch

nielsvanvelzen committed:
Update authorization documentation

nielsvanvelzen committed:
Merge pull request #1559 from jellyfin/nvv-dev-authorization-update

Update authorization documentation

anultravioletaurora committed:
Create authentication.md

anultravioletaurora committed:
Add details for username and password

Update Quick connect details with write up from the kotlin SDK

anultravioletaurora committed:
Update authentication.md

anultravioletaurora committed:
add quick connect details

anultravioletaurora committed:
grammatical fixes

anultravioletaurora committed:
Add links to API documentation

anultravioletaurora committed:
fix sidebar position with dev docs

anultravioletaurora committed:
linting

nielsvanvelzen committed:
Merge pull request #1565 from anultravioletaurora/developers

Add Authentication Developer Documentation

PeachesMLG committed:
Added documentation for Sync play

PeachesMLG committed:
Linting

nielsvanvelzen committed:
Merge pull request #1629 from PeachesMLG/added-syncplay-documentation

Added documentation for Sync play

anultravioletaurora committed:
Add playlist documentation

anultravioletaurora committed:
Merge branch 'jellyfin:developers' into developers

anultravioletaurora committed:
linting

anultravioletaurora committed:
Merge branch 'developers' of github.com:anultravioletaurora/jellyfin.org into developers

anultravioletaurora committed:
update sidebar position

anultravioletaurora committed:
linting, typos

anultravioletaurora committed:
Add description of `isPublic` to the Update docs

nielsvanvelzen committed:
Merge pull request #1670 from anultravioletaurora/developers

Add Playlists Developer Documentation

anultravioletaurora committed:
Add websockets documentation

Update sidebar layout for dev docs

anultravioletaurora committed:
capitalization, linting

anultravioletaurora committed:
spell check

nielsvanvelzen committed:
Merge branch 'master' into developers

anultravioletaurora committed:
update wording

nielsvanvelzen committed:
Merge pull request #1727 from anultravioletaurora/developers

Add Dev Docs for WebSockets

BotBlake committed:
fix direct-donations reference

BotBlake committed:
remove /content 🗿

BotBlake committed:
remove file extension (?)

BotBlake committed:
fix other faulty references

nielsvanvelzen committed:
Merge pull request #1739 from BotBlake/develop

fix direct-donations reference

andrewrabert committed:
Mitigate pull_request_target privilege escalation

Hotfix — replaces pull_request_target with pull_request to stop
granting write permissions and secrets to fork PRs. Some workflows
will break; can be fixed properly later.

anultravioletaurora committed:
Add WebSocket Message Types, Playing Audio Guide

anultravioletaurora committed:
Revert "Add WebSocket Message Types, Playing Audio Guide"

This reverts commit 7f8f850ecdd8de55b65fce869c32a294d6fd8945.

nielsvanvelzen committed:
Merge branch 'master' into developers

nielsvanvelzen committed:
Fix merge 🫠

nielsvanvelzen committed:
Merge branch 'master' into developers
